.github/workflows/ci-cd.yaml

```diff
@@ -0,0 +1,98 @@
+name: Deploy A2rchi
+run-name: ${{ github.actor }} deploys A2rchi
+on: [push]
+jobs:
+  deploy-dev-system:
+    runs-on: ubuntu-latest
+    env:
+      SSH_AUTH_SOCK: /tmp/ssh_agent.sock
+    steps:
+      # boilerplate message and pull repository to CI runner
+      - run: echo "🎉 The job was automatically triggered by a ${{ github.event_name }} event."
+      - uses: actions/checkout@v3
+      - run: echo "The ${{ github.repository }} repository has been cloned to the runner."
+
+      # setup SSH 
+      - name: Setup SSH
+        run: |
+          mkdir -p /home/runner/.ssh/
+          echo "${{ secrets.SSH_PRIVATE_KEY_MDRUSSO }}" > /home/runner/.ssh/id_rsa_submit
```

Comment:
This is just any private key which gets us into the A2rchi service account, correct?

Reply:
Yes correct, so in theory I can replace this with a different key that works w/A2rchi.